From 33042b0af9d4139959a1b9a7179dd3843ea00f0d Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Mon, 24 Sep 2018 17:18:26 +0200
Subject: [PATCH] update-ca: fix build with newer musl

musl removed SYMLINK_MAX define[1]. Use PATH_MAX instead for symlink
target.

[1]: http://git.musl-libc.org/cgit/musl/commit/?id=767f7a1091af3a3dcee2f7a49d0713359a81961c
---
 update-ca.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/update-ca.c b/update-ca.c
index 641fd64..7bb4f1b 100644
--- a/update-ca.c
+++ b/update-ca.c
@@ -196,7 +196,7 @@ static void proc_localglobaldir(const char *fullpath, struct hash *h, int tmpfil
 
 static void proc_etccertsdir(const char* fullpath, struct hash* h, int tmpfile_fd)
 {
-	char linktarget[SYMLINK_MAX];
+	char linktarget[PATH_MAX];
 	ssize_t linklen;
 
 	linklen = readlink(fullpath, linktarget, sizeof(linktarget)-1);
-- 
2.19.0

